python/cuml/__init__.py

```diff
@@ -117,6 +117,7 @@
 
 # Version configuration
 __version__ = "23.12.00"
+__git_commit__ = "HEAD"
```

Comment:
I think it would be better to use an empty string or None as default. HEAD does actual have context-sensitive meaning and would be wrong in many cases, whereas an empty string would clearly indicate that this value is _not set_ in the given scenario.
```suggestion
__git_commit__ = ""
```

Reply:
That's true, there's no guarantee that just because you aren't in a package built using our scripts that you are actually in the git repo. I can change this.